tiledb/cloud/taskgraphs/types.py

```diff
@@ -43,6 +44,38 @@
 """Type returned from an array query."""
 
 
+class Layout(enum.Enum):
+    """The layout of a TileDB query."""
+
+    ROW_MAJOR = "R"
```

Comment:
@ihnorton @nguyenv 

I know I originally used these layout types but should we sync with python enums?

Reply:
If there’s already enums for this in the base Python library I would be thrilled to use them.

Reply:
Or, since there aren’t at the moment (from what I can tell), put them here for now, and then later upstream them, replacing this version with an alias.